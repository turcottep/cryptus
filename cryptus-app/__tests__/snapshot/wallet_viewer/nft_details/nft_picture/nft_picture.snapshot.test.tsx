import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import NftPicture from "../../../../../components/wallet_viewer/nft_details/nft_picture/nft_picture";

describe("<NftPicture />", () => {
  it("should display properly", async () => {
    const testURL = "/images/cryptopunk.png";
    const testDescription = "CryptoPunk #IFORGOR";

    const tree = renderer
      .create(<NftPicture image_url={testURL} description={testDescription} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
