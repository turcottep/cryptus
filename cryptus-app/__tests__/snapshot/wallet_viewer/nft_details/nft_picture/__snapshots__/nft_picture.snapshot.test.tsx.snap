// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NftPicture /> should display properly 1`] = `
<div>
  <img
    alt="CryptoPunk #IFORGOR"
    src="/images/cryptopunk.png"
  />
</div>
`;
